Shuttle (New) Camcorder

Bartholemew-Dingleberry opened the pull request:
<img width="517" height="624" alt="Camcordia" src="https://github.com/user-attachments/assets/58d87ab3-1d58-4deb-8fee-2b49b1f75fff" />
<img width="138" height="149" alt="Camcordia silouhette" src="https://github.com/user-attachments/assets/789fbd6a-44cc-4d48-a211-1ad9c114774b" />
<img width="128" height="96" alt="Camcorder" src="https://github.com/user-attachments/assets/cf71fb9c-fb9c-4faa-9ccd-09a7f43faf4b" />



<!-- Guidelines: https://docs.spacestation14.io/en/getting-started/pr-guideline -->

## About the PR
<!-- What did you change? -->this is the all fancy and new..... news ship, the camcorder! wave to the camera!!!
<!-- If this is a code change, summarize at high level how your new code works. This makes it easier to review. -->

## Why / Balance
<!-- Discuss how this would affect game balance or explain why it was changed. Link any relevant discussions or issues. -->
the lack of big news ship has been felt by all 3 reporter players, with a crew monitor to track down certain people, this big ship is meant to give those with grander plans for hiring crews for large scale reporting an adequate home, as it can fit inside it 3 people! yes 3!! yeah!!!!
## Technical details
<!-- Summary of code changes for easier review. -->
it has some solars, 3 docks, 3 news consoles and 2 cameras with 3 boards for movable cameras as spares, a crew monitor to track down certain people, oh and a lot of folding chairs and tables and some floodlights! yeah!!!!
and a button to turn off the engines in the bridge room (not pictured but mapped)
## How to test
<!-- Describe a procedure to test this feature, along with expected output/behavior. -->
buy the ship and use it?
## Media
<!-- Attach media if the PR makes ingame changes (clothing, items, features, etc).
Small fixes/refactors are exempt. Media may be used in SS14 progress reports with credit. -->

## Requirements
<!-- Confirm the following by placing an X in the brackets [X]: -->
- [X] I have read [CONTRIBUTING.md](https://github.com/new-frontiers-14/frontier-station-14/blob/master/CONTRIBUTING.md) and and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [x] I have added media to this PR or it does not require an ingame showcase.
- [X] I have reviewed the [Ship Submission Guidelines](https://frontierstation.wiki.gg/wiki/Ship_Submission_Guidelines) if relevant.
- [X] I confirm that the content in this PR is my own work, and/or is properly attributed to the original author(s).
<!-- You should understand that not following the above may get your PR closed at maintainer’s discretion -->

## Breaking changes
<!-- List any breaking changes, including namespaces, public class/method/field changes, prototype renames; and provide instructions for fixing them. -->

**Changelog**
<!-- Add a Changelog entry to make players aware of new features or changes that could affect gameplay.
Make sure to read the guidelines and take this Changelog template out of the comment block in order for it to show up.
Changelog must have a :cl: symbol, so the bot recognizes the changes and adds them to the game's changelog. -->
<!--
:cl:
- add: a new news ship has been added, the Camcordia! or well her Prototype, i'd like someone to come and see what it's missing
- remove: Removed fun!
- tweak: Changed fun!
- fix: Fixed fun!
-->

Bartholemew-Dingleberry committed:
the first try at adding a commit for the news ship, Camcordia!

Bartholemew-Dingleberry committed:
removed the power consumption field as i don't know how to find that out easily right now

Bartholemew-Dingleberry committed:
replaced the Disposals with a EVA storage, as the rest of the rooms really didn't need any disposals

Bartholemew-Dingleberry committed:
added the MV wires i somehow forgot like a silly

Bartholemew-Dingleberry committed:
also forgot the engines.....

Bartholemew-Dingleberry committed:
readded the power consumption into the guidebook discription

Bartholemew-Dingleberry commented:
i only now realised that Frontier has a naming scheme of naming things after names (legman/pulsar/ceres/etc) or doers (pathfinder/etc) so it'll probably need renaming to either the camcorder or the camcord

the-hivequeen commented on Resources/Prototypes/_NF/Guidebook/shipyard.yml:
```diff
@@ -40,6 +40,7 @@
   - ShipyardKestrel
   - ShipyardKilderkin
   - ShipyardLegman
+  - ShipyardCamcordia
```
These are alphabetical, shove this up between Caladrius and Camper.

Bartholemew-Dingleberry replied:
thank you, should be fixed after i merge the next commit

the-hivequeen commented on Resources/Prototypes/_NF/Guidebook/shipyard.yml:
```diff
@@ -242,6 +243,11 @@
 
+- type: guideEntry
+  id: ShipyardCamcordia
+  name:guide-entry-shipyard-camcordia
```
```suggestion
  name: guide-entry-shipyard-camcordia
```

the-hivequeen commented on Resources/Prototypes/_NF/Shipyard/camcordia.yml:
```diff
@@ -0,0 +1,41 @@
+  shuttlePath: /Maps/_NF/Shuttles/camcordia.yml
+  guidebookPage: ShipyardCamcordia
+  class:
+  - Service
```
```suggestion
  - Civilian
```

Bartholemew-Dingleberry replied:
yeah i forgot it was called civilian, should be fixed by next commit

the-hivequeen commented:
Hey there! Did a first look at the shuttle. It looks like there were a few issues keeping the ship from showing up in the shipyard and guidebook, but the above fixes should remedy that.

There seems to be some underlying mechanical issues that should be addressed first. I'd recommend looking at the power/atmos cheat sheets that Arimah/Alk have pinned in the mapping-general Discord channel. At first glance it looks like your SMES needs rewired with a cable terminal and the MV wire still needs laid. On the atmos side, swap your gas mixer out with the [On] version to have it start functional and run 'fixgridatmos' on the shuttle before saving to have it start with air. When running power cables and pipes it is preferred not to have them go under walls unless absolutely necessary, it makes it harder for players to make adjustments to their shuttles. Also maybe add some thrusters. ;)

Feel free to ping me on discord if you have any questions. Once you get the baseline systems functioning I'd love to give a more in depth review of the shuttle itself.

Bartholemew-Dingleberry commented on Resources/Locale/en-US/_NF/guidebook/guides.ftl:
```diff
@@ -78,6 +78,7 @@ guide-entry-shipyard-kestrel = Kestrel
 guide-entry-shipyard-kilderkin = Kilderkin
 guide-entry-shipyard-lantern = Lantern
 guide-entry-shipyard-legman = Legman
+guide-entry-shipyard-camcordia = Camcordia
```
thank you, should be fixed after i merge the next commit

Bartholemew-Dingleberry commented on Resources/Prototypes/_NF/Guidebook/shuttle_maps.yml:
```diff
@@ -306,6 +306,17 @@
+  components:
+  - type: Sprite
+    sprite: _NF/Guidebook/shuttle_maps/128x96.rsi
+    state: camcordia
```
how would i make a map? i would rather go all the way and not make a half measure

Bartholemew-Dingleberry committed:
made all the atmos pipes and wires run on floors without walls on top of them, also fixed all the missplacements of the shipyard code,

Bartholemew-Dingleberry committed:
i totaly didn't forget to multitool the scrubbers and vents what are you talking about?

Bartholemew-Dingleberry committed:
2 none glass tiles made glass near the camcordia's right side office

Bartholemew-Dingleberry committed:
added some windows to the small wooden office

Bartholemew-Dingleberry committed:
re-added the Uranium wall/ Green screen i accidentally replaced with normal wall

Bartholemew-Dingleberry committed:
removed some stairs i accidentaly placed multiple of in one tile, removed one stray rail i accidentaly placed, replaced the movable cameras that were already built into boards and added a flatpack, and added 2 rechargers i somehow forgot, and moved the APC and Air alarm from the dim hallway into an inner wall

Bartholemew-Dingleberry committed:
removed all the pipe manifolds and screwdrivered them so that they'd work without the manifolds

Bartholemew-Dingleberry committed:
added the blue color for the pipes that replaced the pipe manifolds

Bartholemew-Dingleberry committed:
Fixes: fixed the LV wires not reaching the docks, a couple of engines being turned off, a Gyroscope being off, and the air alarm not being connected to the fire locks, added missing flooring below some doors.
New Additions: moved the Shipe console to face north and added a crew monitor on the table in the main news room, added a seccond gyroscope and now the ship has 4 forward thrusters 2 at both sides and 2 backwards thrusters.

Bartholemew-Dingleberry committed:
changed all refrences to the ship from "camcordia" to "camcorder" as that name seemed more appropriate

Bartholemew-Dingleberry committed:
added some files i forgot to add

Bartholemew-Dingleberry committed:
added the final file that wasn't tracked

Bartholemew-Dingleberry committed:
added a color coded area map of the shuttle into the Guidebook

Bartholemew-Dingleberry committed:
added a button to turn off engine power with a little dim light bulb to show if it's on or off

Bartholemew-Dingleberry committed:
fixed the guids.ftl having camcordia instead of camcorder

Bartholemew-Dingleberry committed:
fixed none transperant immage and added labels to it (thank you 77reye)
